### PR TITLE
#406 create new class titledisplays student

### PR DESCRIPTION
**Pull request by doritrieur**
please review

### CODE REVIEW

File: src/app/components/navbar/navbar.component.ts [hunk 1]
```diff
@@ -79,4 +81,10 @@ export class NavbarComponent implements OnInit, OnDestroy {
   private removeLeadingSlash(s = '') {
     return s.replace(/^\/+/g, '');
   }
+
+  //@doritrieur in order to get the first part of the path and compare to the route content.
+  private getFirstPathToken(s = '') {
```
Comment (eran1205) [inline src/app/components/navbar/navbar.component.ts]: I think that we already have a mechanism of routing and there we know where are we. Please consult with Celine Zittoun for help. 
The problem with this solution is that if someday someone will change the path, the title won't fit...

Comment (eran1205) [reply src/app/components/navbar/navbar.component.ts]: Hi Dorit,

Any update about this?

Comment (dimkinv) [inline src/app/components/navbar/navbar.component.ts]: can you please change `s` to a more meaningful name?

Comment (doritrieur) [reply src/app/components/navbar/navbar.component.ts]: updated s to "domainString"

**Comment by ulmanA** [comment]
The code has been refactored
Fixed #469

**Pull request merged**